build.gradle

```diff
@@ -7,10 +7,11 @@ def testResultsDir = "${rootProject.buildDir}/test-results"
 buildscript {
 
     ext.kotlin_version = '1.3.60'
-    ext.android_plugin_version = '3.5.2'
+    ext.android_plugin_version = '3.6.0-rc03'
     ext.sdkVersion = 28
     ext.minimumSdkVersion = 14
-    ext.databinding_version = '3.2.0'
+    ext.databinding_version = '3.5.3'
+    ext.viewbinding_version = '3.6.0-rc03'
```

Comment:
I separated library version for ViewBinding from one for DataBinding because DataBinding above 3.6.0 needs the dependency of ViewBinding library (`androidx.databinding:viewbinding`).
I think `databinding_version` and `viewbinding_version` can be merged when 3.6.0 stable has been released.

Reply:
3.6.0 has now been released

Reply:
Thanks for the heads up! I already updated the version of viewbinding at 1aace20.
I did not update the version of databinding because of the class declaration change of `ViewDataBinding` [1][2], but should I also update it to 3.6.0 and merge the variables `databinding_version` and `viewdatabinding_version` to one in this pull request?
When I briefly checked behavior of the example app with Android Gradle Plugin 3.5.3 using `library-databinding` with databinding libraries 3.6.0, no apparent problem was found.

[1] https://androidx.tech/artifacts/databinding/databinding-runtime/3.5.3-source/androidx/databinding/ViewDataBinding.java.html
[2] https://androidx.tech/artifacts/databinding/databinding-runtime/3.6.0-source/androidx/databinding/ViewDataBinding.java.html